mlflow/store/tracking/file_store.py

```diff
@@ -531,6 +531,16 @@ def delete_run(self, run_id):
         new_info = run_info._copy_with_overrides(lifecycle_stage=LifecycleStage.DELETED)
         self._overwrite_run_info(new_info, deleted_time=get_current_time_millis())
 
+    def delete_runs(self, run_ids):
+        for run_id in run_ids:
+            run_info = self._get_run_info(run_id)
+            if run_info is None:
+                raise MlflowException(
+                    f"Run '{run_id}' metadata is in invalid state.", databricks_pb2.INVALID_STATE
+                )
+            new_info = run_info._copy_with_overrides(lifecycle_stage=LifecycleStage.DELETED)
+            self._overwrite_run_info(new_info, deleted_time=get_current_time_millis())
```

Comment:
can we try...catch exception for every single run_id deletion ? otherwise one deletion failure causes all following deletion skipped.